tests/test_parallel.py

```diff
@@ -1,11 +1,13 @@
-import unittest, os, multiprocessing, time
+import unittest, os, multiprocessing, time, sys
 from nutils import parallel
 
 canfork = hasattr(os, 'fork')
 
 class Test(unittest.TestCase):
 
   def setUp(self):
+    if sys.platform == 'darwin':
+      self.skipTest('fork is unreliable (in combination with matplotlib)')
     parallel._maxprocs = 3
```

Comment:
Isn't it cleaner to decorate the class with @skipIf? That should work according to https://docs.python.org/3.5/library/unittest.html#skipping-tests-and-expected-failures

Reply:
done